app/models/concerns/patient_reportable.rb

```diff
@@ -9,19 +9,21 @@ module PatientReportable
     scope :with_hypertension, -> { joins(:medical_history).merge(MedicalHistory.hypertension_yes).distinct }
     scope :excluding_dead, -> { where.not(status: :dead) }
 
-    scope :ltfu_as_of, ->(date) do
+    scope :ltfu_as_of, ->(time) do
+      ltfu_at = (time - LTFU_TIME).end_of_month
```

Comment:
`ltfu_at` and `ltfu_as_of` are very similar. How about
```suggestion
     last_bp_after = (time - LTFU_TIME).end_of_month
```

Reply:
ya I struggled w name for this...I'll go w/ your suggestion.